src/azureStorageHelper.ts

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+import * as storage from 'azure-storage';
+import { FileService } from 'azure-storage';
+import * as path from 'path';
+
+const DIRECTORY_NAME = 'ansible-playbooks';
+
```

Comment:
i would just group all these functions into a single class.

Reply:
methods like is like utility, no need of class with constructor.

Reply:
well, if you encapsulated all these functions into a class you could keep information on created fileshare. but right now every time somebody wants to upload a playbook, all the procedure is started from scratch: createFileService etc.

Reply:
fixed.